source/_integrations/sensehat.markdown

```diff
@@ -140,8 +140,35 @@ group:
   kitchen:
     - group.sense_hat
 ```
-
-### Directions for installing on Raspberry Pi Raspbian Based installation:
+## Installation
+### On Home Assistant Operating System
+
+Things got complicated for poor _SenseHAT_ with Home Assistant Operating System. I found finally a way to use it.
+
+- Follow instructions [here](https://www.home-assistant.io/hassio/enable_i2c/) to enable i2c on Home Assistant Operating System
+- Install "Custom deps deployment" add-on. To do so
+    - add ``https://github.com/home-assistant/hassio-addons-development`` to your add-ons repositories
+    - select *Custom deps deployment* and enable it
+- Configure "Custom deps deployment" add-on
```

Comment:
Custom deps is not recommend, we should not document that and/or writing instructions to use that.

Reply:
we can add a warning if you think it is better, but this is the only way to make the integration works. Otherwise, we can remove the integration as it cannot be installed.

Reply:
A warning makes no sense.

If it doesn't work without these workarounds, I suggest filing an issue and get the problem fixed.

Reply:
I am filing an issue in "home-assistant/operating-system" to include support for RTIMUlib. Can we leave it as "workaround" ? or we will have change this text to something like:

You will not be able to use your SenseHat until [issue 953](home-assistant/operating-system/issues/953) will not be solved.

What do you think?

Reply:
we don't document issues.

Reply:
> Custom deps is not recommend, we should not document that and/or writing instructions to use that.
> we don't document issues.
The larger issue with the documentation is listing SenseHat as a supported device when it's not. That is an issue.

No need to be curt with @denics for trying to address the problem. Three tickets have already been filed and closed as stale. He's proposing two reasonable solutions to make this a better experience for your users.

If you honestly don't want to make it work, that's fine -- in that case, please remove SenseHat from your documentation so others don't waste time like I (and @denics) did.

Reply:
I'm sorry to hear about your issues. However, this is how it is. Feel free to contribute a fix @AgalmicVentures

Reply:
Dear @frenck , the issue here is not with the SenseHat integration, I even made a PR to make the integration more universal and I am ready to add a workflow to it. The problem is that HassOS does not provide a required library. In my "solution" I am suggesting a workaround if HassOS is not integrating the required library. I opened a ticket on HassOS to point that out and you moved it to core (I do not understand why).

Reply:
Because integration defines the installation requirements, not the OS. If the workaround can install. 

If it can be installed by the custom deps add-on, it can be handled by defining dependencies in the integration. Both do the same: Installing Python packages.

Reply:
Gee, I sure wish I'd found this thread six hours ago. I wasted a ridiculous amount of time, thinking "Surely if it's in the documentation, the SenseHAT must work?". I appreciate that this is all done by volunteers, but like others have said, it would be ideal if we could just figure out how to get it working again, but if not REMOVE THE DAMN THING FROM THE DOCUMENTATION or list it as deprecated, something to save people like me from wasting their time.